v2/AdminCore.WebApp/Controllers/WorkingFromHome/WorkingFromHomeController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using AdminCore.WebApi.Models.WorkingFromHome;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace AdminCore.WebApi.Controllers
+{
+  [ApiController]
+  [Authorize]
+  [Route("[controller]")]
+  public class WorkingFromHomeController : ControllerBase
+  {
+    private IMapper _mapper;
+    private IEventService _eventService;
+
+    public WorkingFromHomeController(IEventService eventService, IMapper mapper)
+    {
+      _eventService = eventService;
+      _mapper = mapper;
+    }
+
+    [HttpGet]
+    public IList<WorkingFromHomeViewModel> findAll()
```

Comment:
All method names should be begin with a capital letter.

Reply:
All findall isn't a great method name. can you change it to something like GetAllWfhEvents() to even the GetAllWorkFromHomeEvents